README.md

```diff
@@ -27,24 +27,24 @@ The meeting settings for each channel can be configured in the Channel Header Dr
 
 Meeting settings include:
 
-- Schedule Day: Day of the week when the meeting is scheduled.
+- Schedule Day: Days of the week when the meeting is scheduled.
 - Hashtag Format: The format of the hashtag for the meeting date. The date format is based on [Go date and time formatting](https://yourbasic.org/golang/format-parse-string-time-date-example/#standard-time-and-date-formats)
 
 #### Slash Commands to manage the meeting agenda:
 
 ```
-/agenda queue [next-week] message
+/agenda queue [meetingDay] message
 ```
-Creates a post for the user with the given `message` for the next meeting date. The configured hashtag will precede the `message`.
-If `next-week` is indicated (optional), it will use the date of the meeting in the next calendar week. 
+Creates a post for the user with the given `message` for the next meeting date or the specified `meetingDay` (optional). The configured hashtag will precede the `message`.
+The meeting day supports long (Monday, Tuesday), short name (Mon Tue), number (0-6) or `next-week`. If `next-week` is indicated, it will use the date of the first meeting in the next calendar week. 
```

Comment:
Would it make sense to support `next` as well, whether that's the next day this week, or the next week? Would `next-week` even make sense at that point? (i.e. why would you queue for /this/ day next week, if you have a meeting with multiple days on which to queue agenda items)?

Reply:
> Would `next-week` even make sense at that point?

I debated over this myself and found that `next-week` made more sense in the context of one meeting a week like it was originally. That's why I kept the `next-week` and the day of the week separate because saying `queue Monday next-week` seemed like a stretch to me. 
Also, supporting both optional parameter `next-week` and `[weekday]` overcomplicates a bit the usage of the command. We could possibly come back and revisit this once the Autocomplete branch is merged and we can provide better guidance to the user on how to use the command.

Reply:
Seems fine to keep `next-week` so that it's explicit. Thoughts on also supporting `next` for whatever the next meeting is?

Reply:
Oh I see, I thought before it was a typo when I read `next` 🙈

Yes! `next` would also be very helpful, I can easily add that.

Reply:
Now that I think about it, `next` is the default behavior of `agenda queue` given that it will always queue for the next meeting. 

I already implemented the change but does it make sense to have a command flag that behaves the same with/without it?

Reply:
Ah, yes, `next` would conflict with the vanilla `queue`. In my mind, I guess I was thinking `next-day` or something, but yeah the word `next` is definitely ambiguous. 0/5 on which way to go from here: happy to defer to you :)

Reply:
I am thinking we can leave as is and improve it as we get more feedback once in community.